src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/SolutionView.xaml.cs

```diff
@@ -215,5 +216,11 @@ private void SortableColumnHeader_PreviewKeyUp(object sender, KeyEventArgs e)
                 e.Handled = true;
             }
         }
+
+        private void RequestedVersionColumnHeader_Loaded(object sender, RoutedEventArgs e)
+        {
+            var model = DataContext as PackageSolutionDetailControlModel;
+            _versionColumnHeader.Width = model.IsRequestedVisible ? _versionColumnHeader.Width : 0;
```

Comment:
Awesome! So after this event, when the PMUI reappears, the width looks good? And if you expand/reduce other column widths, nothing weird happens?

Reply:
Yeah it works, but I erased the wrong function and pushed haha, it should be only `_versionColumn.Width` because in the header we are only changing the Visibility property